BearlyHumans/squirrelville
# Koala question mark particle when they have something important to say

This PR adds a question mark particle above koalas when they have something important to say (i.e. before initial dialogue and when the player has collected enough acorns).

Wiki: https://github.com/BearlyHumans/squirrelville/wiki/NPC-Interaction

Steps to test:
1. Be on ISLAND scene
2. Go to koala
3. Make sure question mark is only visible before initial dialogue and when the player has collected enough acorns
4. Make sure the question mark isn't visible when the player is speaking with the koala

https://user-images.githubusercontent.com/62001991/140597353-8cf804b9-333a-470b-b276-e0caf472de51.mp4
